client/lib/presales-chat/index.ts

```diff
@@ -38,10 +38,16 @@ function getGroupName( keyType: KeyType ) {
 	}
 }
 
-export function usePresalesChat( keyType: KeyType, enabled = true, skipAvailabilityCheck = false ) {
+export function usePresalesChat(
+	keyType: KeyType,
+	enabled = true,
+	skipAvailabilityCheck = false,
+	skipEligibilityCheck = false
+) {
 	const isEnglishLocale = useIsEnglishLocale();
 	const { canConnectToZendesk } = useChatStatus();
-	const isEligibleForPresalesChat = enabled && isEnglishLocale && canConnectToZendesk;
+	const isEligibleForPresalesChat =
```

Comment:
I'm not sure why/how GitHub is not showing merge conflicts, but I'd assume you'll need to rebase after https://github.com/Automattic/wp-calypso/pull/80075 landed. And possibly coordinate with folks from that PR to make sure it's working for both cases.